keylime/ca_util.py

```diff
@@ -56,8 +56,8 @@
 elif common.CA_IMPL=='openssl':
     from keylime import ca_impl_openssl as ca_impl
 else:
-    raise Exception("Unknown CA implementation: %s"%common.CA_IMPL)
-from M2Crypto import X509, EVP, BIO
+    raise Exception(f'Unknown CA implementation: {common.CA_IMPL}')
+    from M2Crypto import X509, EVP, BIO
```

Comment:
Found the issue making CI Fail.

Line 60 is indented which means the import does not happen (it only happens as part of the exception occurs..."raise Exception"

Just bring the 4 from back 4 spaces and its all good.

Reply:
Looking at it right now :)